specs/_features/eip7594/das-core.md

```diff
@@ -55,6 +55,7 @@ We define the following Python custom types for type hinting and readability:
 | Name | Value | Description |
 | - | - | - |
 | `NUMBER_OF_COLUMNS` | `uint64(FIELD_ELEMENTS_PER_EXT_BLOB // FIELD_ELEMENTS_PER_CELL)` (= 128) | Number of columns in the extended data matrix. |
+| `MAX_EXTENDED_MATRIX_SIZE` | `uint64(MAX_BLOBS_PER_BLOCK * NUMBER_OF_COLUMNS)` (= 768) | The data size of `ExtendedMatrix`. |
```

Comment:
IMO `MAX_EXTENDED_MATRIX_SIZE` is a bit ambiguous and gives the impression that this is the maximum number of bytes in an extended matrix, but it's actually the maximum number of cells.

We could  clarify this in its description, or rename the constant to `MAX_CELLS_IN_EXTENDED_MATRIX` or something. Let me know what you think and I can do the change if you want.

Reply:
@asn-d6 `MAX_CELLS_IN_EXTENDED_MATRIX` sounds good to me. 👍 feel free to update this PR. Thanks!

Reply:
Should be updated now